test/windows/test.sh

```diff
@@ -21,7 +21,6 @@ set -o pipefail
 export PATH="/c/Program Files/Containerd:$PATH"
 REPO_TAR="${REPO_TAR:-"/c/cri.tar.gz"}"
 FOCUS="${FOCUS:-"Conformance"}"
-SKIP="${SKIP:-"portforward"}"
```

Comment:
Keep the SKIP, just make it empty string.

Reply:
Done.